GitTfs.Vs2017/TfsHelper.Vs2017.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.TeamFoundation.Build.Client;
+using Microsoft.TeamFoundation.Client;
+using Microsoft.TeamFoundation.Server;
+using Microsoft.TeamFoundation.VersionControl.Client;
+using Microsoft.VisualStudio.Setup.Configuration;
+using Sep.Git.Tfs.Core.TfsInterop;
+using Sep.Git.Tfs.VsCommon;
+using StructureMap;
+
+namespace Sep.Git.Tfs.Vs2017
+{
+    public class TfsHelper : TfsHelperBase
+    {
+        private const int REGDB_E_CLASSNOTREG = unchecked((int)0x80040154);
+
+        private const string tfsExtensionsFolder =
+            @"Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer";
+
+        protected string TfsVersionString { get { return "15.0"; } }
+
+        public TfsHelper(TfsApiBridge bridge, IContainer container)
+            : base(bridge, container)
+        { }
+
+        protected override bool HasWorkItems(Changeset changeset)
+        {
+            return Retry.Do(() => changeset.AssociatedWorkItems.Length > 0);
+        }
```

Comment:
This method is defined in `TfsHelperVs2012Base`. Would it make sense to have this `TfsHelper` class inherit from `TfsHelperVs2012Base` instead of from `TfsHelper`? The intention for `TfsHelperVs2012Base` was that it would be a good starting point for VS2012 and everything after that.